boards/sc/scobc_a1/doc/index.rst

```diff
@@ -35,7 +35,7 @@ The Space Cubics OBC module 1 provides the following hardware features:
 +-----------+------------+------------------------------------+
 
 The default configuration for the board can be found in the defconfig file:
-:file:`boards/arm/scobc_module1/scobc_module1_defconfig`.
+:file:`boards/arm/scobc_a1/scobc_a1_defconfig`.
```

Comment:
path does not exist, so is probably easier to replace section with hw-directive block

Reply:
yes +1, the board doc page already has a direct link to "Browse board sources" which should be more than enough

Reply:
Yes, I replaced. Thanks!